app/views/publish/providers/schools_check/show.html.erb

```diff
@@ -0,0 +1,41 @@
+<%= render PageTitle.new(title: "support.providers.schools.check") %>
+<% content_for :before_content do %>
+  <%= govuk_back_link_to(new_publish_provider_recruitment_cycle_school_path) %>
+<% end %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop">
+    <%= form_with(model: @school_form, url: publish_provider_recruitment_cycle_check_school_path(recruitment_cycle_year: @provider.recruitment_cycle_year), method: :put, local: true) do |f| %>
+
+    <fieldset class="govuk-fieldset">
+      <legend class="govuk-fieldset__legend govuk-fieldset__legend--l"> <span class="govuk-caption-l"><%= t("support.providers.schools.new") %> - <%= @provider.decorate.name_and_code %> </span>
```

Comment:
For the non support check page i think we just show the text `Add school` and not with the addition of the provider's name at the end (that's only in support)

Reply:
Done